### PR TITLE
[DA-3887] Added email cron job for participant cleanup notification.

### PR DESCRIPTION
**Pull request by DaveK-14**
## Description of changes/additions
Adds a cron job to automate an email notifying PTSC that they should begin cleanup of test participants.

### CODE REVIEW

File: rdr_service/config.py
Comment (ayazamlani) [inline rdr_service/config.py]: is this code constant needed? I am not sure if it is being pulled on L925 of rdr_service/offline/main.py

**Review by ayazamlani — COMMENTED** [review]
Do we need to write any tests for this new cron job?

File: tests/cron_job_tests/test_offline_app_setup.py
Comment (ayazamlani) [inline tests/cron_job_tests/test_offline_app_setup.py]: shouldn't the mock_checker be getting called at least once since self.send_cron_request('PTSCTestParticipantCleanupRequest') is working and the response is coming back as "success": "true" ? I think it should be getting called but I could be mistaken here

Comment (DaveK-14) [reply tests/cron_job_tests/test_offline_app_setup.py]: I thought that as well, but could not get it to agree. It is getting the correct output from the program, which is all I really wanted. the assert on mock_checker is to prevent circleci from reporting an unused variable since mock_checker is passed in when the test is called.

Comment (kenny-skaggs) [reply tests/cron_job_tests/test_offline_app_setup.py]: I think the timing of the mock is causing this. The `ptsc_test_participant_cleanup_request` method is getting replaced when the test starts, but by the time the mock is applied we've already set up the offline flask app in the setUp method (in the `from rdr_service.offline.main import app` line). So `setUp` processes the offline main.py file, and while doing that still has access to the real definition of the `ptsc_test_participant_cleanup_request` method. Only after that is the mock applied (replacing the method in the offline's main module, but not replacing the handle that the API has for the real definition of the method).

Maybe a target for mocking is to mock the `EmailService.send_email` method and check that it's called (and, even better, something about what was sent matches up with the wording of the reminder email).

Comment (DaveK-14) [reply tests/cron_job_tests/test_offline_app_setup.py]: I have the EmailService.send_email in a block that only runs on prod to avoid sending emails during testing, so it wouldn't be called. I can add the message to the return and assert on that as well, but the closest thing I can think of to check with the mocker in that case would be maybe the Email constructor?

Comment (kenny-skaggs) [reply tests/cron_job_tests/test_offline_app_setup.py]: Yeah, that sounds like it could work.

File: rdr_service/offline/main.py
Comment (kenny-skaggs) [inline rdr_service/offline/main.py]: Importing and using `relativedelta` from the `dateutil.relativedelta` module could be helpful here. `date = datetime.today() + relativedelta(months=1)` would be a little more concise, and would increment the year when needed too.

**Review by kenny-skaggs — CHANGES_REQUESTED** [review]
Please merge the recent changes from devel into this branch (to get the Safety check to pass).

**Comment by kenny-skaggs** [comment]
> Please merge the recent changes from devel into this branch (to get the Safety check to pass).

Nevermind, you're already on it :D

**Review by kenny-skaggs — APPROVED** [review]

**Comment by ayazamlani** [comment]
please get 2 approvals before merging

**Pull request merged**
